wallet/api/openrpc.json

```diff
@@ -5,6 +5,82 @@
     "title": "Wallet API"
   },
   "methods": [
+    {
+      "name": "client.check_transaction",
+      "summary": "Check a transaction or a bundle of transactions and send them to the network.",
+      "description": "This method checks a transaction or a bundle of transactions against the network and send them to the it.\n\nThe network does not add them to a block. What happens with the transaction is up to the third-party application.\n\nThe user has to review the transaction.",
```

Comment:
```suggestion
      "description": "This method sends a transaction or a bundle of transactions to the network to verify their validity.\n\nThe network does not add them to a block. What happens with the transaction is up to the third-party application.\n\nThe user has to review the transaction after this step.",
```

Reply:
@ValentinTrinque please double-check this in particular

Reply:
It's good @candida-d

Reply:
This has been corrected. Thanks!